### PR TITLE
Create Like entity in model

### PR DESCRIPTION
**Pull request by YaroslavDehtiar**

### CODE REVIEW

File: mentionserver/src/main/java/com.mention/model/Post.java [hunk 1]
```diff
@@ -43,6 +43,12 @@
   @OneToMany(mappedBy = "post")
   private List<Comment> comments;
 
+  @OneToMany(mappedBy = "liked")
+  private List<Like> likes;
+
+  @OneToMany(mappedBy = "post")
+  private List<Post> post;
```
Comment (zozich) [inline mentionserver/src/main/java/com.mention/model/Post.java]: Mistake?

File: mentionserver/src/main/java/com.mention/model/User.java [hunk 2]
```diff
@@ -43,6 +43,14 @@
   @JsonIgnoreProperties(value = {"user1"})
   private List<Chat> chats;
 
+  @OneToMany(mappedBy = "liked")
+  @JsonIgnoreProperties("liked")
+  private List<Like> likedUser;
```
Comment (zozich) [inline mentionserver/src/main/java/com.mention/model/User.java]: Do you really need it here?

File: mentionserver/src/main/java/com.mention/model/User.java [hunk 3]
```diff
@@ -43,6 +43,14 @@
...
+
+  @OneToMany(mappedBy = "post")
+  @JsonIgnoreProperties("post")
+  private List<Post> likedPost;
```
Comment (zozich) [inline mentionserver/src/main/java/com.mention/model/User.java]: It's a collection, call it `likedPosts`

**Comment by upastushyna** [comment]
task implement and merged in https://github.com/upastushyna/Mention/pull/54

**Pull request closed without merging**